frontend/resourcebundles/src/main/java/com/virtuslab/gitmachete/frontend/resourcebundles/GitMacheteBundle.java

```diff
@@ -37,11 +39,67 @@ private GitMacheteBundle() {}
 
   @I18nMakeFormat
   public static String getString(@PropertyKey(resourceBundle = BUNDLE) String key) {
-    return instance.getString(key);
+    String string = instance.getString(key);
+    return replaceMnemonicAmpersand(string);
   }
 
   @I18nMakeFormat
   public static @Untainted String getNonHtmlString(@PropertyKey(resourceBundle = BUNDLE) String key) {
     return instance.getString(key);
   }
+
+  /**
+   * This method has been inspired by {@link com.intellij.BundleBase#replaceMnemonicAmpersand}.
+   * It is required to correctly process the mnemonics.
+   */
+  private static String replaceMnemonicAmpersand(String value) {
```

Comment:
We do not process ampersand mnemonics as we should (at least not in all cases) 😅 

I realised it while I created `MyGitNewBranchDialog` and saw `&` characters in the message strings. Then I compared our bundle class with git4Idea bundle class and found `BundleBase`. 

Options here: 
1. To keep this PR smaller - I'd rather keep the current approach for now and 
  a) just leave it as good enough
  b) try to extend `BundleBase` (or `DynamicBundle`) in `GitMacheteBundle` later
    - probably that's smth we not only can but actually should do 
    - the first disadvantage of this solution is that the resourcebundle module will need a dependency on intellij IIUC
2. To keep PR smaller - I could remove the ampersands from the string bundles  for now. And maybe add it later if we implement `BundleBase` or smth.

@PawelLipski , wdyt?

Reply:
Huh actually... do you ever use the ampersand mnemonics yourself? 🤔 I don't remember ever using them, if I use some action frequently then I add a regular shortcut. So... (2.) sounds like a reasonable solution.

Also... what's the problem with **not** processing the mnemonics? did they stop working (assuming no `0x1B` character is added) with some version of IntelliJ? 🤔

Reply:
> Huh actually... do you ever use the ampersand mnemonics yourself? 

yes, sometimes